docs/build-ballerina-from-source.md

```
@@ -2,9 +2,9 @@
 
 ## Overview
 
-[Building the complete Ballerina distribution](#building-the-complete-ballerina-distribution) provides you access to all the main features of Ballerina such as the runtime, corresponding tools, standard library modules etc.
+[Building the complete Ballerina distribution](#building-the-complete-ballerina-distribution) gives you access to all main features of Ballerina, including the runtime, tools, standard library modules, and more.
 
->**Info:** However, if you need just a plain Ballerina language build with only the basic language features and the JBallerina Java (Java Introp) API in it, you can [build only the Ballerina Runtime with the tools](#building-only-the-ballerina-runtime-with-the-tools)
+>**Info:** However, if you need just a plain Ballerina language build with only the basic language features and the JBallerina Java (Java Interop) API in it, you can [build only the Ballerina Runtime with the tools](#building-only-the-ballerina-runtime-with-the-tools)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)